resources/views/stats/stats.blade.php

```diff
@@ -0,0 +1,32 @@
+@extends('layouts.app')
+
+@section('title'){{__('stats')}} @endsection
+
+@section('content')
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-8">
+                <h4>{{strtr(__('stats.personal'), [
+                        ':fromDate' => $from->format('d.m.Y'),
+                        ':toDate' => $to->format('d.m.Y')
+                    ])}}</h4>
+                <hr/>
+                <div class="row">
+                    <div class="col-md-6 mb-4">
+                        @include('stats.includes.chart_companies')
+                    </div>
+                    <div class="col-md-6 mb-4">
+                        @include('stats.includes.chart_categories')
+                    </div>
+                    <hr/>
+                    <div class="col-12 mb-4">
```

Comment:
col-12 sind eh in einer neuen Row, dann kann man sie auch in diese verschieben.

Reply:
Sehe ich keinen Sinn drin, außer, dass wir einen zusätzlich div haben.